src/sleep/sleep.c

```diff
@@ -52,41 +52,31 @@ static int write_hibernate_location_info(const HibernateLocation *hibernate_loca
         assert(hibernate_location);
```

Comment:
you could even make this wording stronger in the commit msg. it is inherently a *good* thing to override the resume offset in the kernel to zero, to undo possibly old half-written settings fully.